src/common_interface/integrator_utils.jl

```diff
@@ -123,7 +123,7 @@ end
 
 function handle_callback_modifiers!(integrator::IDAIntegrator)
     IDAReInit(integrator.mem, integrator.t, integrator.u, integrator.du)
-    DiffEqBase.initialize_dae!(integrator)
+    DiffEqBase.initialize_dae!(integrator, IDADefaultInit())
```

Comment:
why the default here?

Reply:
I thought that after a callback the default initialization was the one we wanted. Is it not?

Reply:
I don't recall if it was using the user-chosen one or the default.

Reply:
OrdinaryDIffEq just calls `integrator.reeval_fsal = true` here. Previously, Sundials would just call `DiffEqBase.initialize_dae!(integrator)` which is now spelled `DiffEqBase.initialize_dae!(integrator, IDADefaultInit())` (since now the initialization is customizable.